docs/index.md

```diff
@@ -0,0 +1,109 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The Payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK allows for client-side approval of an order with the user's card information.
+
+## Eligibility
+```
+# Placeholder: Product Eligibility
+what kind of merchants are we targetting?
+what features do we offer right now/what features are we missing?
+```
+
+## Integration methods
+
+- [Card Fields UI Integration](#card-fields-ui-integration)
+- [Fully Customizable Integration](#fully-customizable-integration)
+
+## Card Fields UI Integration
+
+Accept card payments using our card fields UI
+
+```
+# Placeholder: Include images of card fields
+```
+
+**Requirements:**
+Card fields UI offers a low effort integration where you can use our card fields UI, and our card fields will handle ...
+
+## Fully Customizable Integration
+
+Process card payments with PayPal Payments SDK using your own UI.
+
+### Steps
+
+#### 1. Create an order
+Call `v2/checkout/orders` to create an order an obtain an order ID
+
+```
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <access_token>' \
+--data-raw '{
+    "intent": "CAPTURE",
+    "purchase_units": [
+        {
+            "amount": {
+                "currency_code": "USD",
+                "value": "5.00"
+            }
+        }
+    ]
+}'
+```
+
+#### 2. Approve the order using Payments SDK:
+
+```swift
+import PaymentsSDK
+
+// Create a config object using your clientID obtained from PayPal application portal
+let config = CoreConfig(clientID: "<CLIENT_ID>", evironment: .sandbox)
+
+// Create a CardClient to approve an order with a card payment method
+let cardClient = CardClient(config: config)
+
+// Create a card object
+let card = Card(
+    number: "4111111111111111",
+    expirationMonth: "01",
+    expirationYear: "25",
+    securityCode: "123"
+)
+
+// Create a CardRequest and approve the order using buyer's card when buyer submit their card info.
+let cardRequest = CardRequest(orderID: "<ORDER_ID>", card: card)
```

Comment:
I would create an order at this step. 
The default simple integration that we have talked about with @demerino and others was that the order creation is upon a "button" click. "Creating Order" upfront like it is here to me is an optimization not the default way to integrate particularly as not everyone has all payment method with us so they don't really want to create the order until they think it's worth while and that is only evident once the buyer clicks on a specific payment button and clicks submit.

Reply:
Also note that for "card processing" specifically we can just authorizeOrder or captureOrder without an implicit approveOrder but my recollection is that we did the below to maintain consistency with other payment methods. 

The way the api works is you can do 

1. Single Step (Create Order + Confirm Order + Authorize/Capture) - in a single call by passing card information as part of the order call 

```
"intent": "CAPTURE",
    "purchase_units": [
        {
            "amount": {
                "currency_code": "USD",
                "value": "5.00"
            }
        }
    ]
    "payment_source" : {
    	"card" : {
    		"number":"41111111111111"
    		"expiry":"2023-12"
    		"securityCode":"212"
    	}

    }
```

2. Three Steps
> Create Order
> Confirm Order
> Authorize/Capture Order

Reply:
I believe we still need to spec out the default simple integration but I see that we need separate auth and capture flows. Step 3 in that case would be considered _optional_. I can tweak the docs a bit to reflect this.

Reply:
I don't think I am fully following this. What @rsdighe76 is describing sounds like doing hook into allowing order creation client side which at one point we decided we didn't want to do currently. Has this changed? We _do_ have create order as a client side option on NXO and it added a _ton_ of models since there are a number of options. If we do decide to go this route we should probably decide what is most important to merchants from day one and potentially add other models as merchants request them to keep the SDK light.

Reply:
@jaxdesmarais yeah I believe we agreed in principle to server-side order creation to narrow the scope, but we don't have a formal spec for the default simple integration AFAIK.

Reply:
Cool cool, that makes sense! Just wanted to make sure I wasn't misremembering about not doing create order client side. 😄